src/Controller/UserController.php

```diff
@@ -42,6 +42,7 @@ public function list(): string
             }
         }
         $finalList = array_unique($finalList, SORT_REGULAR);
+        var_dump($finalList);
```

Comment:
No we dont share our var_dump()

Reply:
I delete the var_dump in UserController.